src/index.html

```diff
@@ -17,11 +30,354 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/styles.scss"
```

Comment:
The stylesheet is linked with a .scss file extension. Ensure that the SCSS is compiled to CSS, as browsers do not natively support SCSS files.